cfme/configure/settings.py

```diff
@@ -23,109 +25,173 @@ class TimeProfileAddForm(View):
     save_button = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
     configuration = Dropdown('Configuration')
     table = Table("//div[@id='main_div']//table")
-    save_edit_button = Button('Save')
     cancel_button = Button('Cancel')
     help_block = Text("//span[contains(@class, 'help-block')]")
 
 
-class TimeprofileAddEntities(View):
+class TimeProfileAddEntities(View):
     breadcrumb = BreadCrumb()
     title = Text('//div[@id="main-content"]//h3')
 
 
 class TimeProfileAddFormView(BaseLoggedInPage):
-    timeprofile_form = View.nested(TimeProfileAddForm)
-    entities = View.nested(TimeprofileAddEntities)
+    time_profile_form = View.nested(TimeProfileAddForm)
```

Comment:
Please use `form` here for the view name to keep the view API consistent. See other implementations of nested view forms.